evalai/setup.py

```diff
@@ -0,0 +1,82 @@
+import click
+import sys
+import validators
+
+from click import echo, style
+
+from evalai.utils.auth import (
+    get_host_url,
+    get_user_auth_token_by_login,
+    write_host_url_to_file,
+    write_json_auth_token_to_file,
+)
+
+previous_host_url = get_host_url()
+username_help_message = "Required: Your EvalAI username"
+password_help_message = "Required: Your EvalAI password"
+host_help_message = "Optional: URL of the API host,\
+                    currently set to: {}".format(previous_host_url)
+welcome_text = (
+    """
+                #######                  ###      ###    #######
+                ##      ##   ##   #####  ###     #####     ###
+                #####    ## ##   ##  ##  ###    ##   ##    ###
+                ##        ###   ###  ##  #####  #######    ###
+                #######    #     ### ### #####  ##   ##  #######\n\n"""
+    "Welcome to the EvalAI CLI. Use evalai --help for viewing all the options\n"
+    "CHALLENGE and PHASE placeholders used throughout the CLI are"
+    " for challenge_id\nand phase_id of the challenges and phases."
+)
+
+
+@click.group(invoke_without_command=True)
+@click.option('-p', '--password', type=str, prompt=True,
+              hide_input=True, help=password_help_message)
+@click.option('-u', '--username', type=str, prompt=True,
+              hide_input=False, help=username_help_message)
+@click.option('-h', '--host', type=str, default='', help=host_help_message)
+def ignite(host, username, password):
```

Comment:
@nikochiko I think setup would have been more appropriate here

Reply:
Yeah I was reconsidering it myself.
I thought it could create some conflicts/confusions because setup is used in many places. But it's the best word to use here, so I will make the change.